keyboards/ckeys/handwire_101/keymaps/default/keymap.c

```diff
@@ -0,0 +1,179 @@
+#include "handwire_101.h"
+#include "action_layer.h"
+
+#define _BASE           0  // base layer
+#define _LAYERS         1  // layer of all layers
+#define _MUSIC          2  // music mode
+#define _MUSIC_4_LIFE   3  // music mode until unplugged
+#define _MOUSE          4  // mousekeys
+#define _TERMINAL       5  // terminal
+#define _ADMIN          6  // admin duties
+
+#define _____ KC_TRNS
```

Comment:
QMK defines a keycode for this automatically.

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L30

You can delete Line 12, and replace all instances of  
`_____` (five underscores) with  
`_______` (seven underscores).